docs/BEMAN_STANDARD.md

```diff
@@ -354,6 +325,28 @@ or
 **Status**: [Retired. No longer maintained or actively developed.](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_LIBRARY_MATURITY_MODEL.md#retired-no-longer-maintained-or-actively-developed)
 ```
 
+### **[README.LICENSE]**
+
+**REQUIREMENT**: The `README.md` must contain a section called "License" which specifies the license or licenses used by the library. A non-exhaustive list of examples follows.
```

Comment:
```suggestion
**REQUIREMENT**: Following the library status line and a new line, the `README.md` must have a `License` section.
```

It needs to be consistent to other README.* rules. We need to have a fixed place in the document for this section (it also helps for beman-tidy implementation). Check e.g., README.LIBRARY_STATUS.

>  A non-exhaustive list of examples follows.

I don't agree to add this, we don't do that in the Beman Standard, we try to be always explicit.

Reply:
> ```
> ## License
> 
> beman.foo is dual-licensed under the Apache License v2.0 with LLVM Exceptions and the the Boost Software License 1.0.
> ```
> 
> But I don't think it's useful to enumerate every combination of those here.

Keep only this example in the Standard, and we'll do combinations from beman-tidy checks.